src/lib/services/access-service.ts

```diff
@@ -215,6 +216,19 @@ export class AccessService {
         }
     }
 
+    /**
+     * Returns all roles the user has in the project.
+     * Including roles via groups.
+     * In addition it includes root roles
+     * @param userId user to find roles for
+     * @param project project to find roles for
+     */
+    async getAllProjectRolesForUser(
```

Comment:
Name still good? I read the function name as not returning root roles but the doc string suggests otherwise

Reply:
It does return root roles. The other name `getAllProjectRolesAndRootRolesForUser` was already taken but only returned numbers, might want to look into usage of that other function. For now, keeping this.